modules/installation-configuration-parameters.adoc

```diff
@@ -1372,9 +1377,48 @@ configuring user-defined routing.
 |====
 endif::ash-default[]
 
+ifdef::alibaba[]
+//From: https://github.com/openshift/installer/blob/master/data/data/install.openshift.io_installconfigs.yaml#L20; https://github.com/openshift/openshift-docs/pull/40651/files#r792388476
+
+[id="installation-configuration-parameters-additional-alibaba_{context}"]
+== Additional Alibaba Cloud configuration parameters
+
+Additional Alibaba Cloud configuration parameters are described in the following table:
+
+.Optional {alibaba} parameters
+[cols=".^2m,.^3a,^5a",options="header"]
+|====
+|Parameter|Description|Values
+
+|`platform.alibabacloud.imageID`
+|For compute machines, the image ID that should be used to create ECS instance. If set, the image ID should belong to the same region as the cluster
+|String. 
+
+|`platform.alibabacloud.instanceType`
+|For compute machines, the configuration used when installing on Alibaba Cloud.
+|String. For example `ecs.g6.large`.
+
+|`platform.alibabacloud.systemDiskCategory`
+|For compute machines, the category of the system disk.
+|String, for example "", "cloud_efficiency", "cloud_essd".
+
+|`platform.alibabacloud.systemDiskSize`
+|For compute machine, the size of the system disk in gibibytes (GiB). The minimum is `120`.
+
+|`platform.alibabacloud.zones`
+|For compute machine, list of availability zones that can be used.
+|String.
+
+|====
+
+endif::alibaba[]
```

Comment:
The above fields belong to "**.platform.alibabacloud.defaultMachinePlatform**" instead, please see below: 
```
$ openshift-install explain installconfig.platform.alibabacloud
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <object>
  AlibabaCloud is the configuration used when installing on Alibaba Cloud.

FIELDS:
    defaultMachinePlatform <object>
      DefaultMachinePlatform is the default configuration used when installing on Alibaba Cloud for machine pools which do not define their own platform configuration.

    privateZoneID <string>
      PrivateZoneID is the ID of an existing private zone into which to add DNS records for the cluster's internal API. An existing private zone can only be used when also using existing VPC. The private zone must be associated with the VPC containing the subnets. Leave the private zone unset to have the installer create the private zone on your behalf.

    region <string> -required-
      Region specifies the Alibaba Cloud region where the cluster will be created.

    resourceGroupID <string>
      ResourceGroupID is the ID of an already existing resource group where the cluster should be installed. If empty, the installer will create a new resource group for the cluster.

    tags <object>
      Tags additional keys and values that the installer will add as tags to all resources that it creates. Resources created by the cluster itself may not include these tags.

    vpcID <string>
      VpcID is the ID of an already existing VPC where the cluster should be installed. If empty, the installer will create a new VPC for the cluster.

    vswitchIDs <[]string>
      VSwitchIDs is the ID list of already existing VSwitches where cluster resources will be created. The existing VSwitches can only be used when also using existing VPC. If empty, the installer will create new VSwitches for the cluster.

$ openshift-install explain installconfig.platform.alibabacloud.defaultMachinePlatform
KIND:     InstallConfig
VERSION:  v1

RESOURCE: <object>
  DefaultMachinePlatform is the default configuration used when installing on Alibaba Cloud for machine pools which do not define their own platform configuration.

FIELDS:
    imageID <string>
      ImageID is the Image ID that should be used to create ECS instance. If set, the ImageID should belong to the same region as the cluster.

    instanceType <string>
      InstanceType defines the ECS instance type. eg. ecs.g6.large

    systemDiskCategory <string>
      Valid Values: "","cloud_efficiency","cloud_essd"
      SystemDiskCategory defines the category of the system disk.

    systemDiskSize <integer>
      SystemDiskSize defines the size of the system disk in gibibytes (GiB).

    zones <[]string>
      Zones is list of availability zones that can be used. eg. ["cn-hangzhou-i", "cn-hangzhou-h", "cn-hangzhou-j"]

$

Reply:
@mburke5678 Except the fields under platform.alibabacloud.defaultMachinePlatform, I would suggest to also list the fields under platform.alibabacloud, in the table "Additional Alibaba Cloud configuration parameters". WDYT?

Reply:
@mburke5678 WDYT